src/LazyOperatorAritmetic.jl

```diff
@@ -0,0 +1,107 @@
+
+function QuantumOpticsBase.:+(a::LazyTensor{B1,B2},b::LazyTensor{B1,B2}) where {B1,B2}
+    LazySum(a,b)
+end
+
+function QuantumOpticsBase.:-(a::LazyTensor{B1,B2},b::LazyTensor{B1,B2}) where {B1,B2}
+    LazySum([1,-1],[a,b])
```

Comment:
Does this still work if you use tuples instead of lists? Asking about it because tuples can be made non-allocating, while Julia can not do that to lists yet. If it is impossible, we should probably file a bug with QuantumOpticsBase

Reply:
It is indeed possible to use Tuples instead. I have now changed this in a new commit.